src/components/DayPicker.jsx

```diff
@@ -54,7 +54,9 @@ const YEAR_SELECTION_TRANSITION = 'year_selection';
 
 const propTypes = forbidExtraProps({
   ...withStylesPropTypes,
-
+  // forked additional props
+  forceRender: PropTypes.bool,
```

Comment:
I’m confused why this is needed - can you not force a render by passing a different object down?